.gitignore

```diff
@@ -14,4 +14,7 @@
 .glide/
 
 .idea/
-*.iml
\ No newline at end of file
+*.iml
+
+#vendor files
+vendor
```

Comment:
Remove this change, we don't use `vendor` ever.

Reply:
removing, have a look at other comments and reply, will fix shortly right after